theano/sandbox/gpuarray/tests/test_dnn.py

```diff
@@ -171,7 +171,7 @@ def test_pooling():
         raise SkipTest(dnn.dnn_available.msg)
 
     # 'average_exc_pad' is disabled for versions < 4004
-    if dnn.version() < 4004:
+    if dnn.version(False) < 4004:
```

Comment:
As a parameter is not obvious for a function called version, maybe adding the keyword argument would be good.
`dnn.version(raises=False)`